package.json

```diff
@@ -31,7 +31,7 @@
     "test:flat:build": "yarn test:flat:build:ui && yarn test:flat:build:tests && yarn test:flat:build:locales",
     "test:flat:build:tests": "node test/integration/index.js",
     "test:flat:build:states": "node development/genStates.js",
-    "test:flat:build:locales": "mkdirp dist/chrome && cp -R app/_locales dist/chrome/_locales",
+    "test:flat:build:locales": "mkdir -p dist/chrome && cp -R app/_locales dist/chrome/_locales",
```

Comment:
I do generally think we should keep our development scripts cross-platform; most of the other npm scripts that rely upon Bash are CI scripts, which doesn't need to be cross-platform at all.

But these are just the integration tests, so who cares. This is fine.